sklearn/tree/export.py

```diff
@@ -733,6 +735,10 @@ def export_graphviz(decision_tree, out_file=None, max_depth=None,
         Number of digits of precision for floating point in the values of
         impurity, threshold and value attributes of each node.
 
+    fontname : str,optional (default='helvetica')
+        Set output dot file font
+        optional font name "sans" or other correct font
```

Comment:
Are you trying to suggest `"sans" may have better language support."?

Reply:
yes.

sorry,am I using the wrong word again?

Reply:
Could you make the line up to 79 characters?

```
Set the font used in the output dot-file.
(I am unsure of what you mean with the second sentence)
The default font is 'helvetica' and can be replaced with any available font (maybe?)
```

Reply:
![demo](https://user-images.githubusercontent.com/3999134/61164525-9bb3f000-a548-11e9-99ce-c7613bcdb8e6.PNG)

use the default if it is normal


![demo5](https://user-images.githubusercontent.com/3999134/61164641-dbc7a280-a549-11e9-964a-53b97da9e886.PNG)
if it is not normal,the default value will not be used

Reply:
> Could you make the line up to 79 characters?
> 
> ```
> Set the font used in the output dot-file.
> (I am unsure of what you mean with the second sentence)
> The default font is 'helvetica' and can be replaced with any available font (maybe?)
> ```

I have modified it.